### PR TITLE
Claim payout from bounty list

### PR DESCRIPTION
**Pull request by ekowalsk**
This PR adds Claim button to the bounty list, which allows beneficiary to claim the payout. The button is visible only if among accounts exists beneficiary, otherwise it's not rendered. Clicking the button performs the claim transaction from the beneficiary account.

### CODE REVIEW

File: packages/page-bounties/src/Bounty.tsx [hunk 1]
```diff
@@ -56,6 +57,14 @@ function Bounty ({ bestNumber, bounty, className = '', description, index }: Pro
         <td><DueBlocks dueBlocks={blocksUntilUpdate} /></td>
         <td>{beneficiary ? <AddressSmall value={beneficiary} /> : EMPTY_CELL}</td>
         <td><DueBlocks dueBlocks={blocksUntilPayout} /></td>
+        <td>
+          {beneficiary && blocksUntilPayout &&
+          <BountyClaim
+            beneficiaryId={beneficiary}
+            index={index}
+            payoutDue={blocksUntilPayout}
+          />}
```
Comment (jacogr) [inline packages/page-bounties/src/Bounty.tsx:66]:
```suggestion
          {beneficiary && blocksUntilPayout && (
            <BountyClaim
              beneficiaryId={beneficiary}
              index={index}
              payoutDue={blocksUntilPayout}
            />
          )}
```

File: packages/page-bounties/src/BountyClaim.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+function BountyClaim ({ beneficiaryId, index, payoutDue }: Props) {
+  const { t } = useTranslation();
+  const { api } = useApi();
+  const { allAccounts, hasAccounts } = useAccounts();
+
+  const isBountyClaimable = useCallback(() => isClaimable(allAccounts, beneficiaryId, payoutDue), [allAccounts, beneficiaryId, payoutDue]);
+
+  return hasAccounts && isBountyClaimable()
```
Comment (jacogr) [inline packages/page-bounties/src/BountyClaim.tsx:28]: You probably want this a memo? (aka the call is already one when used)

File: packages/page-bounties/src/BountyClaim.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    ? <TxButton
```
Comment (jacogr) [inline packages/page-bounties/src/BountyClaim.tsx:29]: Would just recommend wrapping inside `(...)` (as per the previous comment)

File: packages/page-bounties/src/helpers/isClaimable.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// Copyright 2017-2020 @polkadot/app-council authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { AccountId } from '@polkadot/types/interfaces';
+
+import BN from 'bn.js';
+
+export const isClaimable = (accounts: string[], beneficiary: AccountId, payoutDue: BN): boolean => {
```
Comment (jacogr) [inline packages/page-bounties/src/helpers/isClaimable.ts:8]: Prefer `export function isClaimable (...) { ... }`

**Review by jacogr — COMMENTED** [review]
Some small comments, overall looks spot-on.

**Comment by ekowalsk** [comment]
Thank you for the really fast response! 😀  I've applied your suggestions

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**